doc/SAI-Proposal-PoE.md

```diff
@@ -0,0 +1,591 @@
+PoE support for SAI
+
+| Title       | PoE                    |
+| ----------- | ---------------------- |
+| Authors     | Daniela Murin, Marvell |
+| Status      | In review              |
+| Type        | Standards track        |
+| Created     | 02/25/2024             |
+| SAI-Version | 1.13                   |
+
+# Overview
+
+Power over Ethernet, or PoE, describes any of several standards that pass electric power along with data on twisted-pair Ethernet cabling. This allows a single cable to provide both a data connection and enough electricity to power networked devices such as wireless access points (WAPs), IP cameras and VoIP phones.
+
+# Proposal
+
+The current SAI does not support PoE. We will implement PoE in a way that any vendor can provide their own PoE hardware and software solution.
+
+## Abbreviations and Information
+
+| **Term** | **Definition**                                                                              |
+| -------------- | ------------------------------------------------------------------------------------------------- |
+| PoE            | Power over Ether                                                                                  |
+| PSE            | Power Sourcing Equipment                                                                          |
+| PD             | Powered Device, can power the device through an PD-capable Ethernet port                          |
+| PoE Port       | Ethernet port that can deliver power and info to devices, located on the front panel of the board |
+
+# Key Ideas behind PoE
+
+1. PoE provides a hardware solution for providing a data connection and electricity to power connected devices
+2. The PoE software delivers information about the board and its operational statuses
```

Comment:
The PoE software provides an interface to configure(program) and query the PoE controller and its internal peripherals such as ports etc.

Reply:
Updated